src/SFA.DAS.AssessorService.Application.Api.Client/Clients/StandardVersionClient.cs

```diff
@@ -39,12 +39,31 @@ public async Task<StandardVersion> GetStandardVersionByStandardUId(string standa
                 return await RequestAndDeserialiseAsync<StandardVersion>(request, $"Could not find the standard {standardUId}");
             }
         }
+
+        public async Task<IEnumerable<StandardVersion>> GetEpaoRegisteredStandardVersions(string epaoOrgId)
```

Comment:
If I'm being pedantic "epaoOrgId" should be "epaOrgId" as in End Point Assessor Organisation Id.

Reply:
Have renamed the variable where it was incorrect 👍